UKHomeOffice/nmsw-ui
# NMSW-450 Update 'Account created' page content

# Ticket

NMSW-450

----

## AC

Update content for account created page to match the prototype

----

## To test

- Run app
- Go to localhost:3000
- Go to /create-account/account-created
- > You should see a confirmation banner but with no user specific information
- Go back to the landing page
- Go through the create account process until the success page
- > The Account created page should contain content that displays the full name, company name and email you signed up with

----

## Notes
